src/main/kotlin/com/hedvig/underwriter/service/QuoteServiceImpl.kt

```
@@ -106,11 +113,15 @@ class QuoteServiceImpl(
         return quoteRepository.find(completeQuoteId)
     }
 
-    override fun getQuoteFromMemberId(memberId: String): QuoteDto? {
-        val quote = quoteRepository.findByMemberId(memberId)
-        return QuoteDto.fromQuoteDto(quote!!)
+    override fun getSingleQuoteForMemberId(memberId: String): QuoteDto? {
```

this'll throw an NPE rather than return null if the quote doesn't exist no?